tools/generator/test/games/test.yml

```diff
@@ -39,6 +39,13 @@ struct:
       - [field_int, int, "Integer field"]
       - [field_int_arr, int array, "Integer array field"]
       - [field_str_arr, simple_struct array, "Structure array field"]
+  -
```

Comment:
Maybe it's better to have this in an other PR, since it might break the CI for other languages for something not linked to Java?

Reply:
I’d agree if it did break the CI. But since it does not (and frankly, the yaml already generate a struct of list of struct, here it is just a struct of struct, so I don’t see how it would break stuff), I don’t really see the point in making an other PR.

Here it is just to see the codepath in the java generator especially made for struct of struct.